Podfile

```diff
@@ -8,10 +8,10 @@ workspace 'WordPress.xcworkspace'
 
 ## Pods shared between all the targets
 def shared_with_all_pods
-  pod 'CocoaLumberjack', '~> 3.2.0'
-  pod 'FormatterKit/TimeIntervalFormatter', '~> 1.8.1'
+  pod 'CocoaLumberjack', '3.2.0'
+  pod 'FormatterKit/TimeIntervalFormatter', '1.8.2'
   pod 'NSObject-SafeExpectations', '0.0.2'
-  pod 'UIDeviceIdentifier', '~> 0.1'
+  pod 'UIDeviceIdentifier', '~> 0.4'
```

Comment:
Automattic-Tracks-iOS has a pod dependency of `pod 'UIDeviceIdentifier', '~> 0.4'` so I can't go any higher than this without updating that podspec (which I eventually will do).